bridge/opencensus/internal/otel2oc/span_context_test.go

```diff
@@ -45,12 +58,24 @@ func TestSpanContextConversion(t *testing.T) {
 				TraceOptions: octrace.TraceOptions(0),
 			},
 		},
+		{
+			description: "trace state should be propagated",
+			input: trace.NewSpanContext(trace.SpanContextConfig{
+				TraceID:    trace.TraceID([16]byte{1}),
+				SpanID:     trace.SpanID([8]byte{2}),
+				TraceState: tsOtel,
+			}),
+			expected: octrace.SpanContext{
+				TraceID:      octrace.TraceID([16]byte{1}),
+				SpanID:       octrace.SpanID([8]byte{2}),
+				TraceOptions: octrace.TraceOptions(0),
+				Tracestate:   tsOc,
+			},
+		},
 	} {
 		t.Run(tc.description, func(t *testing.T) {
 			output := SpanContext(tc.input)
-			if output != tc.expected {
-				t.Fatalf("Got %+v spancontext, expected %+v.", output, tc.expected)
-			}
+			assert.Equal(t, tc.expected, output)
```

Comment:
```suggestion
			assert.Equal(t, tc.expected, output)

			// Ensure the otel tracestate and oc tracestate has the same header output
			_, ts := httpFormatOc.SpanContextToHeaders(tc.expected)
			assert.Equal(t, tc.expectedTracestate, ts)
			assert.Equal(t, tc.expectedTracestate, tc.input.TraceState().String())

			// The reverse conversion should yield the original input
			input := oc2otel.SpanContext(output)
			assert.Equal(t, tc.input, input)
```

To make sure the header output of tracestate is intact after the convert.